apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -44,6 +44,12 @@ export function AiGeneratePage({ resLocals }: { resLocals: Record<string, any> }
                 </button>
               </form>
               <div id="generation-results"></div>
+              <br />
+              <div>
+                <a href="${resLocals.urlPrefix}/ai_generate_question_jobs">
+                  <button class="btn btn-primary">Previous Generation Jobs</button>
+                </a>
```

Comment:
Why are you nesting a `<button>` inside an `<a>`? You can use `<a>` directly.

Reply:
ugh, forgot that you can attach button formatting to an <a> tag. changed in a133e9b738f1eb22f09f6fa83561847be451b8eb.